app/Models/Like.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Like extends Model
+{
+    use HasFactory;
+
+    protected $table = 'likes';
+    protected $primaryKey = 'id';
+
+    public function user()
+    {
+        return $this->belongsTo(User::class, 'user_id');
+    }
+    public function book()
+    {
+        return $this->belongsTo('App\Models\Book', 'book_id');
```

Comment:
use App\Models\Book; vào rồi dùng khai báo kiểu này
```suggestion
        return $this->belongsTo(Book:class);
```